README.md

```diff
@@ -216,4 +216,11 @@ constructorIO.trackConversion(itemName: "Fashionable Toothpicks", customerID: "1
 
 // Track when items are purchased
 constructorIO.trackPurchase(customerIDs: ["123-AB", "456-CD"], revenue: 34.49, orderID: "343-315")
+
+// New way to track when items are purchased (supported in v2.5.5 and above)
```

Comment:
Need to make sure that this version number doesn't get changed when versioning/deploying the repo